packages/workflow/docs/WorkflowExample.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+// import { Button } from "@kaizen/button"
+// import VisibleIcon from "@kaizen/component-library/icons/visible.icon.svg"
+import { Button } from "@kaizen/button"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import {
+  Workflow,
+  WorkflowExit,
+  WorkflowFooter,
+  WorkflowHeader,
+  WorkflowProps,
+} from "../"
```

Comment:
Probably need to have a chat about this naming, cause it's going to be a bit awkward especially when from the consumer side it would look like this:

```
import {
  Workflow,
  WorkflowExit,
  WorkflowFooter,
  WorkflowHeader,
  WorkflowProps,
} from "@kaizen/workflow" // workflow is implied already
```

And in 99% of cases, it looks like this:
```
import { Workflow } from "@kaizen/workflow"
```

Reply:
Oh this is just a stand-in while I was building out all the components and testing how they slot together. The final one here would be just using the Workflow component from `Worflow.tsx`. Or did you mean if someone was going to use the individual components that should use dot notation, ie: Workflow.Header?

Reply:
Yeah the latter, but not sure the dot notation applies cause it doesn't have a hard dependency on `Workflow` to work eg. context

Reply:
> I think just to be complete, we should have a `Main` component to encapsulate the styles and `main` landmark 😄

Sounds like a way to nudge users to ensure the main landmark is used 👍